src/openrct2/actions/RideSetStatus.hpp

```diff
@@ -73,9 +74,16 @@ DEFINE_GAME_ACTION(RideSetStatusAction, GAME_COMMAND_SET_RIDE_STATUS, GameAction
 
         if (_status != ride->status)
         {
-            if (_status == RIDE_STATUS_TESTING)
+            if (_status == RIDE_STATUS_SIMULATING && (ride->lifecycle_flags & RIDE_LIFECYCLE_BROKEN_DOWN))
             {
-                if (!ride_is_valid_for_test(ride, _status == RIDE_STATUS_OPEN, 0))
+                // Simulating will force clear the track, so make sure player can't cheat around a break down
+                res->Error = GA_ERROR::DISALLOWED;
+                res->ErrorMessage = STR_HAS_BROKEN_DOWN_AND_REQUIRES_FIXING;
+                return res;
+            }
+            else if (_status == RIDE_STATUS_TESTING || _status == RIDE_STATUS_SIMULATING)
+            {
+                if (!ride_is_valid_for_test(ride, _status, 0))
```

Comment:
Might be a good time to turn the last parameter into an actual bool since this PR touches both lines, its not used anywhere else.